invisible_cities/cities/buffy.py

```diff
@@ -112,9 +112,10 @@ def buffy(files_in     , file_out   , compression      , event_range,
                                           extract_tminmax               ,
                                           bin_pmt_wf                    ,
                                           bin_sipm_wf                   ,
+                                          buffer_calculation            ,
                                           fl.branch("event_number"      ,
                                                     evtnum_collect.sink),
-                                          buffer_calculation            )   ,
+                                          fl.sink(lambda x: x))         ,
```

Comment:
Again, you won't need a cryptic `/dev/null` sink if you do the previous step without branching off.